### PR TITLE
New Package: libsass 3.3.6

### PR DESCRIPTION
**Pull request by arete**
Signed-off-by: Gerardo Di Iorio <arete74@gmail.com>

### CODE REVIEW

File: srcpkgs/libsass/template [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Template file for 'libsass'
+pkgname=libsass
+version=3.3.6
+revision=1
+wrksrc="$pkgname-$version"
```
Comment (Duncaen) [inline srcpkgs/libsass/template]: thats the default

**Comment by arete** [comment]
ok , i have removed the line 5.
regards

**Comment by Duncaen** [comment]
`usr/lib/libsass.a` should be moved to the `-devel` package too.
And I would remove the `(library)` from `short_desc`.

**Comment by Vaelatern** [comment]
Do the move with `vmove usr/lib/*.a`

**Comment by arete** [comment]
i have update pull request

File: srcpkgs/libsass/template [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+	autoreconf  -i
+}
+
+
+post_install() {
+	vlicense LICENSE
+}
+
+libsass-devel_package() {
+	depends="libsass>=${version}_${revision}"
```
Comment (ebfe) [inline srcpkgs/libsass/template]: Could use `depends="${sourcepkg}>=${version}_${revision}"`

**Comment by arete** [comment]
ok i have modify the commit wiut yours suggestion

**Pull request merged**